doc/src/fix_bond_react.rst

```diff
@@ -581,10 +593,14 @@ Default
 """""""
 
 The option defaults are stabilization = no, prob = 1.0, stabilize_steps = 60,
-reset_mol_ids = yes, custom_charges = no
+reset_mol_ids = yes, custom_charges = no, molecule = off
 
 ----------
 
 .. _Gissinger:
 
 **(Gissinger)** Gissinger, Jensen and Wise, Polymer, 128, 211 (2017).
+
+.. _Gissinger:
```

Comment:
there must not be the same anchor target twice. "make html" should print a warning about this.

Reply:
@akohlmey on it